backend/schema/corpora_schema.md

```diff
@@ -169,37 +111,25 @@ organism|String
 organism\_ontology\_term\_id|NCBITaxon term
 layer\_descriptions|Mapping from {layer\_name: layer\_description, ...} Each description is free text, though one layer must be described as "raw".
```

Comment:
Are there minimum requirements for the layer descriptions? We've come across "final" matrices that have gone through Seurat integration and ambient noise removal through SoupX . Is the goal for layer description to capture anything that has been done to the matrix expression numbers?

Reply:
We haven't really been able to come up with requirements other than urging people to be kind of descriptive. There's just too much diversity in the things people do to the data.